src/main/scripts/src/app/core/dto/attribute.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+export interface Attribute {
+
+  name: string;
+  fullName: string;
+  count: number;
+  constraints: object[];
```

Comment:
The `name` and `fullName` are always the same because we don't have nested attributes now. So you shoudn't remove `fullName` attribute, I'm sure we will use it in the future.